blocks/columns/columns.css

```diff
@@ -1,33 +1,67 @@
+.section > div.columns-wrapper {
+  margin-top: 100px;
+  margin-bottom: 50px;
+}
+
+.columns {
+  font-family: var(--body-book-font-family);
+  font-size: var(--body-font-size-s);
+}
+
 .columns > div {
   display: flex;
   flex-direction: column;
+  margin-bottom: 100px;
+}
+
+.columns > div > div {
+  order: 1;
+}
+
+.columns > div:last-of-type {
+  margin-bottom: 0;
 }
 
 .columns img {
   width: 100%;
 }
 
-.columns > div > div {
-  order: 1;
+.columns h2 {
+  margin-top: -10px;
 }
 
 .columns > div > .columns-img-col {
   order: 0;
+  flex-basis: 22%;
 }
 
 .columns > div > .columns-img-col img {
   display: block;
 }
 
-@media (min-width: 900px) {
+@media (max-width: 991px) {
```

Comment:
Would prefer only `min-width` unless there is a strong case for max-width. I will leave it as your call here.